gnus-recent.el

```diff
@@ -148,15 +149,15 @@ article list is the article we're currently looking at."
            (gnus-summary-refer-article message-id)))))))
 
 (defun gnus-recent--action (recent func)
-  "Find message-id and group arguments from RECENT, call FUNC on them.
+  "Find `message-id' and group arguments from RECENT, call FUNC on them.
 Warn if RECENT can't be deconstructed as expected."
   (pcase recent
     (`(,_ . (,message-id ,group . ,_))
      (funcall func message-id group))
     (_
      (message "Couldn't parse recent message: %S" recent))))
 
-(defun gnus-recent--open-article (recent)
+(defun gnus-recent--open-article-orig (recent)
```

Comment:
Just delete the -orig function. 

The old one is in git, and `org-gnus.el` is included at least in emacs 26.1 and 25.2. I would have used it if I knew about it :-)

Reply:
OK